server/jobs/helpers.js

```diff
@@ -56,5 +56,6 @@ export async function executeEmailAutomation(emailFunctions, placement) {
     // write automation success to database
   } catch (error) {
     // write automation failure to database
+    console.log(error.message);
```

Comment:
You can remove the `console.logs` after debugging

Reply:
@nzediegwu1 I have made the requested change. Thanks for the feedback